### ISSUE TITLE
stop using WMCore.Credential.Proxy

### ISSUE THREAD

**Issue by belforte** [dmwm/CRABClient#4973]
modity https://github.com/dmwm/CRABClient/blob/master/src/python/CRABClient/CredentialInteractions.py
replacing usage of WMCore Credential class with simple, standalone code, focused on CRAB needs which forks calls to voms-proxy-* and myproxy-* as neeeded.

Already WMCore's Credential.Proxy is de-facto factorized in two parts, one used by CRAB, one by WMCore.
The use of the common abstract class WMCore/Credential is nice, but really not worth.
In any case we use no other credential then VOMS atm, and will transition to tokens which will likely have a different semantic.
Directly exposing myproxy-* commands, will only help code maintenance.

**Comment by belforte** [comment]
simplifications I can do :
- [ ] stop using serverdn becasue myproxy credential name does not depend on server name anymore
- [ ] do not mix voms-proxy and myproxy in same class, just call myproxy* commands in `createNewMyProxy`
- [ ] does it really make sense to worry about group and role here ? when interacting with cmsweb VOMS extenions are ignored.

**Comment by belforte** [comment]
I decided to limit changes, i.e. not touch SubCommand.py and createmyproxy.py (the only files which talks with CredentialInteractions), but  modified the CredentialInteractions.py main class to interact with two simpler objects from the new ProxyInteractions.py file, ne to deal with voms-* and one with myproxy-* instead of a single "proxy" object as in WMCore/Proxy.
At the same time I removed unused/uneeded options and am trying to clarify interfaces, especially with regards to time units. Also use  a better name than "serverDn" for the list of authorized retrievers.
